fmf/base.py

```diff
@@ -641,13 +641,13 @@ def _get_node_raw_data(self, default=None):
             raw_data = raw_data[key]
         return raw_data
 
-    def modify(self, data, method="update"):
+    def modify(self, *data, method="update"):
```

Comment:
Looks fine. Just not sure whether it's better to use `data` or `*data` here. The second has advantage that one could provide multiple dictionaries, not sure how frequent such use case would be. Any other pros? @lukaszachy, any preference here?

Reply:
I can see here these pros and cons:
- data
  - pros
    - update - cleaner it takes exactly one arg
    - pop - pop also takes righ one arg
  - cons
    - cannot pop more values in one command, what i see useful
    - clear command is strange that you have to set value, or put there some default like None, but it may lead to situation that you have to chech value for update or pop

- *data
  - pros:
    - pop - easy to remove more items
    - clear - no defaut value needed and expect no additional value
  - cons:
    - update - little bit strange possibility to update same dict moretimes,is it useful, probably not much from my PoV

Reply:
Personally I like the approach of e.g. [Thread](https://docs.python.org/3/library/threading.html#threading.Thread) - `target=None, args=(), kwargs={}`. Just rename target to method so it is clear that it is name of the dict method as a string.

Reply:
> Personally I like the approach of e.g. [Thread](https://docs.python.org/3/library/threading.html#threading.Thread) - `target=None, args=(), kwargs={}`. Just rename target to method so it is clear that it is name of the dict method as a string.

Thats also not bad, but what I don't like is the syntax then, so that, instead of using `node.modify({a=b, c=d})` you have to use `node.modify(args=({a=b, c=d},))` (yes, you can also have there possibility to check if args is not iterable, use it as one item, but it is more tricky from type annotation then) what is confusing for me. but it is cleaner from perspective, that args, and kwargs are posted to `method`
In fact some tools using this and another uses `*args, **kwargs` and move these explicitly undefined attributes into internally used methods.